mathics/builtin/stats.py

```diff
@@ -0,0 +1,1743 @@
+import sympy
+import sympy.stats
+from mathics.builtin.base import SympyFunction, Builtin
+from mathics.core.expression import Symbol, Complex, Integer, Real
+from mathics.core.convert import from_sympy, sympy_symbol_prefix
+
+
+
+class DistributionParam(object):
+    msg = ''
+
+    @staticmethod
+    def convert(value):
+        return
+
+    @staticmethod
+    def get(value): #gets value for check_too -- todo
+        return
+
```

Comment:
this is to get the param value, to be used by check_too. Is it a simpler way to check parameters?